velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1068,6 +1095,9 @@ void JoinFuzzer::verify(core::JoinType joinType) {
               defaultPlan.plan->outputType(),
               {expected}),
           "Velox and Reference results don't match");
+
+      LOG(INFO) << "Result matches with referenc DB.";
```

Comment:
do we need this log?

Reply:
I added this log because I found it makes it easier to tell whether one iteration verifies with the reference DB.